README.md

```diff
@@ -0,0 +1,83 @@
+# nightfall_dlp
+![nightfalldlp](https://www.finsmes.com/wp-content/uploads/2019/11/Nightfall-AI.png "nightfalldlp")
+### NightfallDLP - a code review tool that protects you from committing sensitive information
+
+NightfallDLP scans your commits for secrets or sensitive information and posts review comments to your code hosting 
+services automatically. It’s intended to be used as a part of your CI to simplify the development process, improve your 
+security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
+
+## Supported CI Services
+### GithubActions
+Example using the NightfallDLP Github Action inside a Github Workflow  
+_(**N.B.** you must use the actions/checkout step before the running the nightfalldlp action in order for it to function properly)_
+```yaml
+name: nightfalldlp
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  run-nightfalldlp:
+    name: nightfalldlp
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo Action
+        uses: actions/checkout@v2
+
+      - name: nightfallDLP action step
+        uses: nightfallai/nightfall_dlp_action@CreateAction
+        env:
+          NIGHTFALL_API_KEY: ${{ secrets.NIGHTFALL_API_KEY }}
+          EVENT_BEFORE: ${{ github.event.before }}
+```
+
+### Action Spec  
+NightfallDLP requires a config file and a few Environment variables in order to run  
+
+**Config File (detectors)**  
+ - place a `.nightfalldlp/` directory within the root of your target repository, and inside it a `config.json` file
+ in which you can configure your detectors (see `Detectors` section below for more information on Detectors)  
+ - inside the `detectors` map the keys are Detector Names and the values are the minimum likelihood for a detector to be triggered 
+
+ - sample `.nightfalldlp/config.json` file
+```json
+{
+    "detectors": {
+        "CREDIT_CARD_NUMBER": "POSSIBLE",
```

Comment:
I'm assuming you must do this filtering in the Action, since the API just always uses POSSIBLE as a threshold today. Should we mention that the minimum threshold we currently support is POSSIBLE, or is that tipping our hand too much?

Reply:
ah good question - i didn't realize we don't support detector thresholds yet. I think it probably makes sense to be clear about that

Reply:
but i actually am not sure i understand what you mean - like the API just ignores whatever thresholds we send over, and returns any findings it finds (in the detector set we provide) w/ POSSIBLE certainty or greater?

Reply:
@henkejosh Correct -- this is the request schema that the API currently supports: https://developer.nightfall.ai/reference#making-your-first-request

We would need to open a PR to update the object schema to add a new field like `minFindingThreshold` to that detector object.

Reply:
OH gotcha, OK cool